carla:
  host: localhost
  port: 2000
  QualityLevel: Low  # Low or Epic
  SynchronousMode: True
  SendNonPlayerAgentsInfo: True
  NumberOfVehicles: 1
  NumberOfPedestrians: 1
  WeatherId: 1
  Framesperepisode: -1 # -1: run episode indefinitely until interrupted
  Episodes: 2

  sensors:
    # camera_front:
    #   SensorType: CAMERA
    #   carla_settings:
    #     PostProcessing: SceneFinal
    #     ImageSizeX: 320
    #     ImageSizeY: 240
    #     FOV: 90
    #     PositionX: 1.80
    #     PositionY: 0
    #     PositionZ: 1.30
    #     RotationPitch: 0
    #     RotationRoll: 0
    #     RotationYaw: 0

    # camera_depth:
    #   SensorType: CAMERA
    #   carla_settings:
    #     PostProcessing: Depth
    #     ImageSizeX: 320
    #     ImageSizeY: 240
    #     FOV: 90
    #     PositionX: 1.80
    #     PositionY: 0
    #     PositionZ: 1.30
    #     RotationPitch: 0
    #     RotationRoll: 0
    #     RotationYaw: 0

    lidar_0:
      SensorType: LIDAR_RAY_CAST
      carla_settings:
        Channels: 40
        Range: 100
        PointsPerSecond: 720000
        RotationFrequency: 10
        UpperFovLimit: 7
        LowerFovLimit: -16
        PositionX: 0.2
        PositionY: 0
        PositionZ: 2.00
        RotationPitch: 0
        RotationYaw: 0
        RotationRoll: 0
